### DISCUSSION TITLE
Store logs in `$XDG_STATE_HOME`

### DISCUSSION THREAD

**Discussion by roadkell** [marlonrichert/zsh-autocomplete#561] [Ideas]
According to [XDG base dir spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html),
> The $XDG_STATE_HOME contains state data that should persist between (application) restarts, but that is not important or portable enough to the user that it should be stored in $XDG_DATA_HOME. It may contain:
>    - actions history (**logs**, history, recently used files, …)
> ...

Also, thanks for the awesome plugin!

**Comment by marlonrichert** [comment]
Done: 784e3a9
